packages/playground/src/utils/utils.ts

```diff
@@ -70,3 +70,44 @@ export const timeDistance = (date1 = 0, date2 = 0, onlyHighestDenomination = fal
     minutes ? minutes + 'm' : hours || days ? '0m' : ''
   } ${seconds}s`
 }
+
+/**
```

Comment:
I already added this util https://github.com/harness/canary/blob/main/apps/gitness/src/pages/pipeline-edit/utils/time-utils.ts#L47

Reply:
Looks like we need the same util in `package/playground` as well.